src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -28,15 +28,15 @@
     <TestWithCore Condition="'$(TestWithCore)' == ''">true</TestWithCore>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TestWithCore)' == 'true'">
+  <PropertyGroup Condition="'$(TestWithCore)' == 'true' And '$(TestAppX)' != 'true'">
```

Comment:
So currently is it correct to assume that when we test UWP, we will run build.cmd /p:TestTFM=netcore50 /p:TestAppX=true ?